### PR TITLE
Specify that the sandbox has an inbuilt _ object from lodash v3:10.1

### PR DESCRIPTION
**Pull request by meenakshi-dhanani**

### CODE REVIEW

File: src/pages/docs/writing-scripts/script-references/postman-sandbox-api-reference.md [hunk 1]
```diff
@@ -756,7 +756,7 @@ The `require` method allows you to use the sandbox built-in library modules. The
 * [cheerio](https://cheerio.js.org/)
 * [crypto-js](https://www.npmjs.com/package/crypto-js)
 * [csv-parse/lib/sync](https://csv.js.org/parse/)
-* [lodash](https://lodash.com/)
+* [lodash](https://lodash.com/) (inbuilt `_` object v3.10.1 exists in the sandbox by default, use require to load the latest version)
```
Comment (DannyDainton) [inline src/pages/docs/writing-scripts/script-references/postman-sandbox-api-reference.md:759]: I would go with `built-in`,  as that would be consistent with the sentence above the list of modules.

Apart from that, it's cool. :D

**Review by jkonrath-postman — CHANGES_REQUESTED** [review]
Hi @meenakshi-dhanani, thanks for the changes!

How about this wording?
(The built-in `_` object v3.10.1 exists in the sandbox by default. Use `require` to load the latest version.)

File: src/pages/docs/writing-scripts/script-references/postman-sandbox-api-reference.md (hunk 1 above)
Comment (meenakshi-dhanani) [reply src/pages/docs/writing-scripts/script-references/postman-sandbox-api-reference.md]: Done

**Review by sowmyaraj92 — APPROVED** [review]
Looks good! 🎉

**Review by DannyDainton — APPROVED** [review]

**Review by jkonrath-postman — APPROVED** [review]
Looks good!
